web/src/modals_util.ts

```diff
@@ -0,0 +1,33 @@
+import {$t} from "./i18n";
+
+export const custom_time_unit_values = {
+    minutes: {
+        name: "minutes",
+        description: $t({defaultMessage: "minutes"}),
+    },
+    hours: {
+        name: "hours",
+        description: $t({defaultMessage: "hours"}),
+    },
+    days: {
+        name: "days",
+        description: $t({defaultMessage: "days"}),
+    },
+    weeks: {
+        name: "weeks",
+        description: $t({defaultMessage: "weeks"}),
+    },
+};
+
+export function get_custom_time_in_minutes(time_unit: string, time_input: number): number {
```

Comment:
I added a new module for these as it didn't seem to fit with what is in `web/src/settings_ui.ts`. But maybe there's a better location for these functions that I didn't think about?

Reply:
There is a `util.ts` file.

Reply:
Makes sense. Updated to move/add any utility functions to `web/src/util.ts`.